integrations/node/workload-executor.js

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+
+const MongoClient = require('../../node-mongodb-native').MongoClient;
+const fs = require('fs');
+const assert = require('assert');
+const omit = require('lodash.omit');
+
+let interrupted = false;
+process.on('SIGINT', () => (interrupted = true));
+
+const executors = {
+  find: (collection, args) =>
+    collection.find(args.filter || {}, omit(args, ['filter'])).toArray(),
```

Comment:
seems like we would want some more validation here, in case someone fat fingered an operation definition